src/nspawn/nspawn.c

```diff
@@ -1018,14 +1011,17 @@ static int parse_argv(int argc, char *argv[]) {
                         assert_not_reached("Unhandled option");
                 }
 
+        if (getenv_bool("SYSTEMD_NSPAWN_SHARE_SYSTEM") > 0)
+                arg_share_system = true;
```

Comment:
In the sense, that we don't really have a way of knowing if people are using it in the wild or not.